elasticsearch/src/main/java/com/gentics/mesh/search/impl/ElasticSearchProvider.java

```diff
@@ -97,7 +97,42 @@ public void start() {
 
 	@Override
 	public ElasticSearchProvider init() {
-		return this;
+		JsonObject settings = null;
+
+        try {
+            settings = client.clusterSettings().sync();
+        } catch (HttpErrorException e) {
+			log.error("Could not determine current cluster settings: {}", e.getMessage(), e);
+
+			return this;
+        }
+
+		Optional<Boolean> settingsDestructiveRequiresName = Optional.ofNullable(settings)
+				.map(o -> o.getJsonObject("persistent"))
+				.map(o -> o.getJsonObject("action"))
+				.map(o -> o.getString("destructive_requires_name"))
+				.map(Boolean::valueOf);
+
+		if (settingsDestructiveRequiresName.isEmpty() || settingsDestructiveRequiresName.get()) {
+			log.info("Updating cluster configuration: destructive_requires_name=false");
+
+			JsonObject update = new JsonObject()
+				.put("persistent", new JsonObject()
+					.put("action", new JsonObject()
+						.put("destructive_requires_name", false)));
+
+            try {
+                settings = client.updateClusterSettings(update).sync();
```

Comment:
Updating the elasticsearch configuration like this feels like a bad idea:
What if the client is not allowed to do this? (init of the provider will log an error message and the setting will not be changed, bad things will follow)
What if elasticsearch instance is recreated and Mesh not restarted? (setting is back to the configured state, bad things will follow)
We do not mention anything of that in our documentation. Silently fiddling with our customers servers is a bad habit.

Why not just documenting, how elasticsearch needs to be configured to properly work with Mesh?